core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/rds/RdsConfigV4Endpoint.java

```diff
@@ -27,68 +27,68 @@
 @Consumes(MediaType.APPLICATION_JSON)
-@Api(value = "/v3/{workspaceId}/rdsconfigs", description = ControllerDescription.RDSCONFIGS_V3_DESCRIPTION, protocols = "http,https")
-public interface RdsConfigV3Endpoint {
+@Api(value = "/v4/{workspaceId}/rdsconfigs", description = ControllerDescription.RDSCONFIGS_V4_DESCRIPTION, protocols = "http,https")
+public interface RdsConfigV4Endpoint {
 
     @GET
     @Path("")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RdsConfigOpDescription.LIST_BY_WORKSPACE, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
             nickname = "listRdsConfigsByWorkspace")
-    Set<RDSConfigResponse> listByWorkspace(@PathParam("workspaceId") Long workspaceId, @QueryParam("environment") String environment,
+    Set<RDSConfigResponse> list(@PathParam("workspaceId") Long workspaceId, @QueryParam("environment") String environment,
             @QueryParam("attachGlobal") Boolean attachGlobal);
 
     @POST
     @Path("")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RdsConfigOpDescription.CREATE_IN_WORKSPACE, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
             nickname = "createRdsConfigInWorkspace")
-    RDSConfigResponse createInWorkspace(@PathParam("workspaceId") Long workspaceId, @Valid RDSConfigRequest request);
+    RDSConfigResponse create(@PathParam("workspaceId") Long workspaceId, @Valid RDSConfigRequest request);
 
     @GET
     @Path("{name}")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RdsConfigOpDescription.GET_BY_NAME_IN_WORKSPACE, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
             nickname = "getRdsConfigInWorkspace")
-    RDSConfigResponse getByNameInWorkspace(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
+    RDSConfigResponse get(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
 
     @DELETE
     @Path("{name}")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RdsConfigOpDescription.DELETE_BY_NAME_IN_WORKSPACE, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
             nickname = "deleteRdsConfigInWorkspace")
-    RDSConfigResponse deleteInWorkspace(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
+    RDSConfigResponse delete(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
 
     @GET
     @Path("{name}/request")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RdsConfigOpDescription.GET_REQUEST_IN_WORKSPACE, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
             nickname = "getRdsRequestFromNameInWorkspace")
-    RDSConfigRequest getRequestFromName(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
+    RDSConfigRequest getRequest(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
 
     @POST
     @Path("testconnect")
```

Comment:
use the operation with _, test_connection

Reply:
i thought we should use only alphanumeric chars

Reply:
Yes, as we discussed earlier, but we've checked the rest api standards with @doktoric and they are suggest this form

Reply:
ok, please update the jira issue and we can follow those standards